src/bootstrap/builder.rs

```diff
@@ -312,7 +312,7 @@ impl<'a> Builder<'a> {
                 tool::RustInstaller, tool::Cargo, tool::Rls, tool::Rustdoc, tool::Clippy,
                 native::Llvm, tool::Rustfmt, tool::Miri, native::Lld),
             Kind::Check => describe!(check::Std, check::Test, check::Rustc),
-            Kind::Test => describe!(test::Tidy, test::Bootstrap, test::Ui, test::RunPass,
+            Kind::Test => describe!(test::Tidy,/* test::Bootstrap,*/ test::Ui, test::RunPass,
```

Comment:
Spurious change. I should investigate...

Reply:
What's this about?

Reply:
Bootstrap tests look for `cc`, which do not exist on Windows and I need to run tests locally to update them.